src/sections/collection/Collection.module.scss

```diff
@@ -13,5 +13,6 @@
 }
 
 .subtext {
+  margin-right: 5px;
```

Comment:
This margin right could be deleted if we add a flex container, see comment below.